src/_base/docker/image/cron/root/cron-run-with-env.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+IFS=$'\n'
+env_vars="$(cat /app/env.sh)"
+/usr/bin/env - "$env_vars" "$@" > /proc/1/fd/1 2> /proc/1/fd/2
```

Comment:
does this definitely work correctly with multiple env vars? with the now added quotes, I believe it would see as
```
/usr/bin/env - "ENV_VAR1=value1 ENV_VAR2=value2" ...
```
rather than:
```
/usr/bin/env - "ENV_VAR1=value1" "ENV_VAR2=value2"
```

This is why I mentioned you may need to tell shellcheck to ignore the lack of quoting

Reply:
yes it works

Reply:
no.. maybe not! I will adjust it

Reply:
I will re-test it to be sure

Reply:
The following cron 

* * * * * /cron-run-with-env.sh env

Gives me this in the logs

~/workspace/github/olp-akeneo feature/OLP-1445-separate-cron-container *6 !1 ?1 ❯ docker logs b500f2b8532a                                                                                                12s
PROCESS_ENDPOINT=http://middleware
KAFKA_HOST=kafka:29092
DB_HOST=mysql
PROCESS_NOTIFICATION_ENDPOINT=pim/tellmeaboutit
DB_PLATFORM=mysql
HOSTNAME=b500f2b8532a
PHPIZE_DEPS=autoconf 		dpkg-dev 		file 		g++ 		gcc 		libc-dev 		make 		pkg-config 		re2c
GPG_KEYS=1729F83938DA44E27BA0F4D3DBDB397470D12172 B1B44D8F021E4E2D6021E995DC9FF8D3EE5AF27F
DB_USER=akeneo
APP_MODE=development
AKENEO_APP_MODE=dev
PHP_EXTRA_CONFIGURE_ARGS=--enable-fpm --with-fpm-user=www-data --with-fpm-group=www-data --disable-cgi
PHP_ASC_URL=https://www.php.net/get/php-7.2.26.tar.xz.asc/from/this/mirror
PHP_CFLAGS=-fstack-protector-strong -fpic -fpie -O2 -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
KAFKA_TOPIC=productUpdate
DB_PASS=akeneo
PWD=/app
HOME=/root
PHP_IDE_CONFIG=serverName=workspace
HOST_OS_FAMILY=darwin
PHP_LDFLAGS=-Wl,-O1 -Wl,--hash-style=both -pie
PHP_INI_DIR=/usr/local/etc/php
PHP_URL=https://www.php.net/get/php-7.2.26.tar.xz/from/this/mirror
PHP_CPPFLAGS=-fstack-protector-strong -fpic -fpie -O2 -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
AKENEO_SECRET=AMjnA6QnWfkZ9s8vB2XXLALXuxZm7fmC
APP_NAME=olp-akeneo
APP_HOST=olp-akeneo.my127.site
ASYNC_NOTIFICATION_ENABLED=1
PHP_VERSION=7.2.26
SHLVL=1
DB_NAME=akeneo
PHP_MD5=
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/app/bin
PHP_SHA256=1dd3bc875e105f5c9d21fb4dc240670bd2c22037820ff03890f5ab883c88b78d
_=/usr/bin/env
PROCESS_ENDPOINT=http://middleware
KAFKA_HOST=kafka:29092
DB_HOST=mysql
PROCESS_NOTIFICATION_ENDPOINT=pim/tellmeaboutit
DB_PLATFORM=mysql
HOSTNAME=b500f2b8532a
PHPIZE_DEPS=autoconf 		dpkg-dev 		file 		g++ 		gcc 		libc-dev 		make 		pkg-config 		re2c
GPG_KEYS=1729F83938DA44E27BA0F4D3DBDB397470D12172 B1B44D8F021E4E2D6021E995DC9FF8D3EE5AF27F
DB_USER=akeneo
APP_MODE=development
AKENEO_APP_MODE=dev
PHP_EXTRA_CONFIGURE_ARGS=--enable-fpm --with-fpm-user=www-data --with-fpm-group=www-data --disable-cgi
PHP_ASC_URL=https://www.php.net/get/php-7.2.26.tar.xz.asc/from/this/mirror
PHP_CFLAGS=-fstack-protector-strong -fpic -fpie -O2 -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
KAFKA_TOPIC=productUpdate
DB_PASS=akeneo
PWD=/app
HOME=/root
PHP_IDE_CONFIG=serverName=workspace
HOST_OS_FAMILY=darwin
PHP_LDFLAGS=-Wl,-O1 -Wl,--hash-style=both -pie
PHP_INI_DIR=/usr/local/etc/php
PHP_URL=https://www.php.net/get/php-7.2.26.tar.xz/from/this/mirror
PHP_CPPFLAGS=-fstack-protector-strong -fpic -fpie -O2 -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
AKENEO_SECRET=AMjnA6QnWfkZ9s8vB2XXLALXuxZm7fmC
APP_NAME=olp-akeneo
APP_HOST=olp-akeneo.my127.site
ASYNC_NOTIFICATION_ENABLED=1
PHP_VERSION=7.2.26
SHLVL=1
DB_NAME=akeneo
PHP_MD5=
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/app/bin
PHP_SHA256=1dd3bc875e105f5c9d21fb4dc240670bd2c22037820ff03890f5ab883c88b78d
_=/usr/bin/env

BUT ... running the crontab under the www-data user seems to prevent the logs from being appended to the root process's stdout, so I will adjust how that works

Reply:
I can't run crontab directly under www-data as it cannot log to stdout of the main process for docker logs to view.  So I have changed the script used in the crontab to run commands as www-data.  Not as good, but we need the logs.

Reply:
yep, makes sense actually, as you're piping the output to the /proc/1, which no non-root user would be able to do. The change to run just the internal task as the user will be good enough still, as that's where the main concern arrises